packages/mobile/test/schemas.ts

```diff
@@ -291,6 +291,14 @@ export const v1Schema = {
   },
 }
 
+export const v2Schema = {
```

Comment:
Added v2 schema to avoid conflicting with some issues in v1 schema migration, which I believe are being fixed in https://github.com/celo-org/celo-monorepo/pull/3791